packages/atlas/src/topic.ts

```diff
@@ -24,4 +25,16 @@ export class Topic extends Base {
   async deleteTopic(id: number) {
     return await this.del(`/api/v1/topic/${id}`);
   }
+
+  async findUserTopics(): Promise<FindUserTopicsApiResponse> {
+    return await this.get(`/api/v1/topic/of/user`);
+  }
+
+  async addUserTopics(topicIds: number[]): Promise<void> {
+    return await this.post(`/api/v1/topic/of/user`, { topicIds });
+  }
+
+  async deleteUserTopics(topicIds: number[]): Promise<void> {
+    return await this.del(`/api/v1/topic/of/user`, { topicIds });
+  }
```

Comment:
To link the frontend with the backend and make it more type-safe. Let's define two types `ITopc.AddUserTopicsApiPayload` and `ITopic.DeleteUserTopicsApiPayload` and use these types here and in the backend.